Library/Homebrew/dev-cmd/boneyard-formula-pr.rb

```diff
@@ -0,0 +1,104 @@
+# Creates a pull request to boneyard a formula.
+#
+# Usage: brew boneyard-formula-pr [options...] <formula-name>
+#
+# Options:
+#   --dry-run:  Print what would be done rather than doing it.
+
+require "formula"
+require "utils/json"
+require "fileutils"
+
+module Homebrew
+  def boneyard_formula_pr
+    formula = ARGV.formulae.first
+    odie "No formula found!" unless formula
+
+    formula_path = formula.path
+    core = formula_path.to_s.include? "homebrew-core"
+    if ARGV.dry_run?
+      ohai "brew update"
```

Comment:
I think for all dry-run blocks it would be useful to print the directory (normally some repository) the following commands would be executed in. Could be a text header using `oh1` saying something like `In <directory>:` or just another `ohai` outputting `cd <directory>`.